docs/source/essentials/http-cache.mdx

```diff
@@ -0,0 +1,101 @@
+---
+title: HTTP Cache
+---
+
+import {MultiCodeBlock} from 'gatsby-theme-apollo-docs';
+
+Apollo Android has 2 different kinds of caches: HTTP cache and normalized cache. The HTTP cache is easier to setup but also has more limitations. This page focuses on the HTTP cache. If you want to deduplicate the storage of your objects and/or notify your UI when your data changes, take a look at the [normalized cache](./normalized-cache) instead.
+
+## Http Cache
+
+To enable HTTP Cache support, add the dependency to your project's build.gradle file. The latest version is
+[![Download](https://api.bintray.com/packages/apollographql/android/apollo/images/download.svg)](https://bintray.com/apollographql/android/apollo-http-cache/_latestVersion)
+
+```kotlin:title=build.gradle
+dependencies {
+  implementation("com.apollographql.apollo:apollo-http-cache:x.y.z")
+}
+```
+
+### Raw HTTP Response Cache
```

Comment:
"response cache" (we use sentence case everywhere for article titles and headings)